### DISCUSSION TITLE
http://layzspa.local/ is not secure

### DISCUSSION THREAD

**Discussion by Bricklin-am** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#735] [General]
I cannot acsess the lay spa server its up and running but chrome , firefox and edge tells me its not safe, I can not find a way around this either so i cannot acess it. How can i fix this ?

**Comment by visualapproach** [comment]
Use http:// not https://
If that's not it, you need to provide more info.

**Reply by Bricklin-am** [reply]
I use http://  same issue when I use the IP adresse instead. I have a WeMos D1 4.0.0 usbc Esp8266 card bought from Aliexpress. I get this error after wifi I setup.

**Reply by visualapproach** [reply]
It's a problem with your network setup. Well, maybe not a problem per se as it's correct in telling you that the website is not secured. But I imagine you have a proxy or something in between that requires https. I'm not a network expert so maybe google is your best friend here. You should have the ESP on the same network as the browser.
